### PR TITLE
Update deps

### LINKED ISSUE #2476: Update to `prost 0.9.0`
Likely involves updating related dependencies such as tonic, which no longer requires (or implements) Sync for bidirectional streaming from v0.5 to 0.6, so also not a straightforward upgrade (which is useful, because it allows to drop the dependency on the sync_wrapper crate and simplify some of our logic)

### PR DESCRIPTION
**Pull request by tiziano88**
Fix dependabot

Fixes #2476

### CODE REVIEW

File: experimental/https_attestation/Cargo.toml [hunk 1]
```diff
@@ -36,4 +37,4 @@ tokio = { version = "*", features = [
   "sync",
   "rt-multi-thread"
 ] }
-tokio-rustls = "*"
+tokio-rustls = "0.22"
```
Comment (rbehjati) [inline experimental/https_attestation/Cargo.toml:40]: Does the `# TODO(#2544)` apply here too?

**Review by rbehjati — APPROVED** [review]
Nice :)

File: experimental/https_attestation/src/main.rs [hunk 2]
```diff
@@ -33,38 +33,38 @@
...
 use anyhow::{anyhow, Context};
+use clap::Parser;
 use futures::FutureExt;
 use log::info;
-use structopt::StructOpt;
 
-#[derive(StructOpt, Clone)]
-#[structopt(about = "HTTPS Attestation")]
+#[derive(Parser, Clone)]
+#[clap(about = "HTTPS Attestation")]
```
[the remark this answers is not in the document]
Comment (tiziano88) [reply experimental/https_attestation/src/main.rs:41]: Good idea, changed everywhere.

File: experimental/https_attestation/Cargo.toml (hunk 1 above)
Comment (tiziano88) [reply experimental/https_attestation/Cargo.toml:40]: May or may not, I hope it will be more obvious when we do the update.

**Review by tiziano88 — COMMENTED** [review]
Thanks for the quick review

**Pull request merged**